script.js

```diff
@@ -1,36 +1,48 @@
 'use strict';
 
 const startButton = document.querySelector('input[value="スタート！"]');
-let printRemainingTime = document.querySelector('#result');
+const stopButton = document.querySelector('input[value="ストップ！"]');
 let countdown;
 
 // 入力された時間を変換
 function fetchDueTime() {
   let due = document.querySelector('input[type="time"]');
-  const dueYear = new Date().getFullYear();
-  const dueMonth = new Date().getMonth();
-  const dueDate = new Date().getDate();
+  let fetchDate = new Date();
+  const dueYear = fetchDate.getFullYear();
+  const dueMonth = fetchDate.getMonth();
+  const dueDate = fetchDate.getDate();
   const dueHours = Number(due.value.slice(0,2));
   const dueMinutes = Number(due.value.slice(3,5));
   const dueSeconds = 0;
   return new Date(dueYear, dueMonth, dueDate, dueHours, dueMinutes, dueSeconds);
```

Comment:
setHours()を使うと短くできそう！

```suggestion
  const dueHours = Number(due.value.slice(0, 2));
  const dueMinutes = Number(due.value.slice(3, 5));

  return new Date().setHours(dueHours, dueMinutes, 0, 0);
```